src/Acuminator/Acuminator.Tests/Tests/StaticAnalysis/EventHandlerModifier/EventHandlerModifierFixerTests.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Acuminator.Analyzers.StaticAnalysis.EventHandlerModifier;
+using Acuminator.Analyzers.StaticAnalysis.PrivateEventHandlers;
+using Acuminator.Analyzers.StaticAnalysis.PXGraph;
+using Acuminator.Tests.Helpers;
+using Acuminator.Tests.Verification;
+using Acuminator.Utilities;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Xunit;
+
+namespace Acuminator.Tests.Tests.StaticAnalysis.EventHandlerModifier
+{
+	public class EventHandlerModifierFixerTests : CodeFixVerifier
+	{
+		protected override CodeFixProvider GetCSharpCodeFixProvider() => new EventHandlerProtectedModifierFix();
+
+		protected override DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer() => new PXGraphAnalyzer(
+				CodeAnalysisSettings.Default
+									.WithRecursiveAnalysisEnabled()
+									.WithStaticAnalysisEnabled()
+									.WithSuppressionMechanismDisabled(),
+				new EventHandlerModifierAnalyzer());
+
+
+		[Theory]
+		[EmbeddedFileData("InvalidHandlerModifier.cs",
+						  "InvalidHandlerModifier_Expected.cs")]
+		public void Test_Invalid_Modifiers_In_Event_Handlers(string actual, string expected)
+		{
+			VerifyCSharpFix(actual, expected);
+		}
+
+		[Theory]
+		[EmbeddedFileData("ContainerWithInterface.cs",
```

Comment:
We usually also write tests on the "Expected" files that contain expected result after code fix. 
Such tests check that no diagnostic is reported.

Reply:
I already have some test cases where there is no change in the "expected" file. Please let me know if you'd like me to improve this further. Thank you.

Reply:
I meant a bit different test, not a test that code fix does not change anything on some file without error.
The "Expected" code file represents the code after the code fix application. We usually test that no diagnostics are found for such files. It looks something like this:
```cs
[Theory]
[EmbeddedFileData("PrivateEvents_Expected.cs")]
public async Task GraphEventHandlers_WithPrivateAccessModifier_AfterFix_NoDiagnostic(string actual) =>
	await VerifyCSharpDiagnosticAsync(actual);
```

Reply:
Ok, I understand now when I read it again :)
I'll add tests like this.

Reply:
Done. Added tests for the five expected files.